src/index.html

```diff
@@ -9,10 +9,62 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="styles/stars.css"
+    />
+    <link
+      rel="stylesheet"
+      href="styles/stars__star.css"
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <body>
```

Comment:
You have a `body` tag inside another `body` tag. This is not valid HTML. There should only be one `body` tag in a HTML document. Please remove the extra `body` tag.